src/vs/workbench/contrib/notebook/browser/view/renderers/webviewPreloads.ts

```diff
@@ -1225,6 +1225,8 @@ async function webviewPreloads(ctx: PreloadContext) {
 
 		viewModel.toggleDragDropEnabled(currentOptions.dragAndDropEnabled);
 
+		document.designMode = 'Off';
```

Comment:
Just in case, should this in a `finally` block of the `try/catch` above. That way we disable design mode even if an error is thrown